aries-site/src/examples/components/pagination/PaginationServerTableExample.js

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
+import { Box, DataTable, Heading, Pagination, Text, Tip } from 'grommet';
+import { StatusCritical } from 'grommet-icons';
+
+const columns = [
+  {
+    property: 'name',
+    header: 'Name',
+    size: 'small',
+    primary: true,
+  },
+  {
+    property: 'rocket',
+    header: 'Rocket',
+    size: 'small',
+    render: datum => <Text key={datum.rocket.name}>{datum.rocket.name}</Text>,
+    align: 'end',
+  },
+  {
+    property: 'success',
+    header: 'Success',
+    size: 'xsmall',
+    align: 'end',
+    render: datum => {
+      if (datum.success === false) {
+        const content = (
+          <Box width={{ max: 'medium' }}>
+            {datum.failures?.map(({ reason }) => (
+              <Text key={reason}>{reason}</Text>
+            ))}
+          </Box>
+        );
+        return (
+          <Tip
+            plain
+            content={content}
+            dropProps={{
+              round: 'medium',
+              pad: 'small',
+              background: 'background-back',
+            }}
+          >
+            <Box>
+              <StatusCritical color="red" />
+            </Box>
+          </Tip>
+        );
+      }
+      return undefined;
+    },
+  },
+];
+
+export const PaginationServerTableExample = () => {
+  const [sort, setSort] = useState({ property: 'name', direction: 'asc' });
+  const [data, setData] = useState([]);
+  const limit = useState(10);
+  const [numberItems, setNumberItems] = useState(0);
+  const [page, setPage] = useState(1);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const query = {
+        options: {
+          populate: [
+            {
+              path: 'rocket',
+              select: { name: 1 },
+            },
+          ],
+          sort: {
+            [sort.property || 'name']: sort.direction || 'asc',
+          },
+          select: ['name', 'success', 'failures'],
+          limit,
+          page,
+        },
+      };
+      fetch('https://api.spacexdata.com/v4/launches/query', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(query),
+      })
+        .then(response => response.json())
+        .then(d => {
+          setPage(d.page);
+          setNumberItems(d.totalDocs);
+          setData(d.docs || []);
+        })
+        .catch(error => console.error('Unable to get data:', error));
+    };
+
+    fetchData();
+  }, [sort, page, limit]);
+
+  return (
+    <Box pad="small">
+      <Heading
+        id="server-side-pagination-heading"
+        level={3}
+        margin={{ bottom: 'small', top: 'none' }}
+      >
+        Launches
+      </Heading>
+
+      <Box>
+        <DataTable
+          aria-describedby="server-side-pagination"
+          columns={columns}
+          data={data}
+          sortable
```

Comment:
Should we be using `sort` here? https://v2.grommet.io/datatable#sort

```suggestion
          sort={sort}
```

Reply:
The pagination example on the DS site [here](https://design-system.hpe.design/components/datatable#paginated) uses the same `sortable` code. Functionality wise there is a small difference in the initial datatable view presented to the user, we can touch base and decide together!